spec/spec_helper.rb

```diff
@@ -1,3 +1,9 @@
+require 'simplecov'
```

Comment:
You could also try putting this inside `bin/rails` per [simplecov instructions](https://github.com/colszowka/simplecov#getting-started). Depending on load order of everything, putting it in spec_helper might miss a few things (that we may or may not actually have coverage on)